include/crm/common/output.h

```diff
@@ -714,6 +714,11 @@ void
 pcmk__html_add_header(xmlNodePtr parent, const char *name, ...)
 G_GNUC_NULL_TERMINATED;
 
+#define PCMK__OUTPUT_SPACER(out, cond) \
+    if(cond) { \
+        out->info(out, "%s", ""); \
+    }
+
```

Comment:
Let's name it something like PCMK__OUTPUT_SPACER_IF() to emphasize it's conditional. Alternatively, would it be worth adding a method out->spacer(out, condition)?

Reply:
I could go either way with it.  We'd been trying to keep the number of basic functions to a minimum, and it's only going to do something on text-based formats for now, so that's two reasons to not.  On the other hand, I generally like doing as little as possible with macros so that would be a reason to do so.  I guess I don't really have any strong opinion.

Reply:
The most recent push has this renamed to PCMK__OUTPUT_SPACER_IF, but I could go another way if needed.